### ISSUE TITLE
Help with Sdr options

### ISSUE THREAD

**Issue by ImWick** [osqzss/gps-sdr-sim#254]
How I can send fake location of a particular device IMEI to particular server or port

**Comment by Mictronics** [comment]
GPS and IMEI are two different things. Like apple and orange.

**Comment by ImWick** [comment]
Yeah I know that, assume there is a device an electronic fare meter with gps integrated in it the gps is sending logs on a particular server which is also available on a web app and the this EFM device id is an unique key for that web app on which basis it receives data
